hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -153,6 +153,8 @@ static boolean isValidSuffix(String suffix) {
   // This suffix and all after that are used for initialization of the various partitions. The unused suffixes lower than this value
   // are reserved for future operations on the MDT.
   private static final int PARTITION_INITIALIZATION_TIME_SUFFIX = 10; // corresponds to "010";
+  // we have max of 4 partitions (FILES, COL_STATS, BLOOM, RLI)
+  private static final List<String> VALID_PARTITION_INITIALIZATION_TIME_SUFFIXES = Arrays.asList("010","011","012","013");
```

Comment:
Should this be dynamically generated based on the number of indexes / partition in the metadata table?  For example, a new index can be added and update to this list may be missed.

Reply:
for 0.14.0, since our MDT partitions are static, I did not want to fix that yet.

Reply:
Makes sense.  We should still track this as we may add new MDT partition in the future.